polaris-react/src/components/Banner/tests/Banner.test.tsx

```diff
@@ -263,31 +263,31 @@ describe('<Banner />', () => {
         const banner = mountWithApp(<Banner />);
 
         const bannerDiv = banner.find('div', {
-          className: 'Banner withinPage',
+          className: 'Box Banner withinPage',
         });
 
         bannerDiv!.trigger('onKeyUp', {
           target: bannerDiv!.domNode as HTMLDivElement,
         });
 
         expect(banner).toContainReactComponent('div', {
-          className: 'Banner keyFocused withinPage',
+          className: 'Box Banner keyFocused withinPage',
         });
       });
 
       it('does not add a keyFocused class onMouseUp', () => {
         const banner = mountWithApp(<Banner />);
 
         const bannerDiv = banner.find('div', {
-          className: 'Banner withinPage',
+          className: 'Box Banner withinPage',
```

Comment:
We can also do `expect.stringContaining('Banner withinPage')` to make these tests less fragile